cfme/services/service_catalogs/ui.py

```diff
@@ -149,8 +149,8 @@ class OrderServiceCatalogView(OrderForm):
     @property
     def is_displayed(self):
         return (
-            self.in_explorer and self.service_catalogs.is_opened and
-            self.title.text == 'Order Service "{}"'.format(self.context['object'].name)
+            self.in_service_catalogs and self.service_catalogs.is_opened and
+            'Service "{}"'.format(self.context['object'].name) in self.title.text
```

Comment:
In latest builds word 'Order' has been removed from the title text hence changed it to this, it is backward compatible.